examples/src/main/scala/feature/GeoCrunch.scala

```diff
@@ -39,7 +39,7 @@ trait GeoCrunch {
   ) = {
     val it = fc.features()
     try {
-      while (it.hasNext) try {
```

Comment:
What was your goal with this try @dwins? The compiler warned me that it did nothing without its own catch or finally clause. Whole loop is wrapped too.